base_contact/README.rst

```diff
@@ -0,0 +1,7 @@
+This module allows you to manage your contacts
+==============================================
+
+Features
+--------
+    - a contact can have several job positions in different companies,
+    - birthdate and nationality can be defined.
```

Comment:
I'was not sure about what the content of 'Installation' and 'Configuration' sections.
Also, there are others contributors to add, in Git I have found TheCloneMaster, Sandy Carter, EL HADJI DEM and Xavier ALT from OpenERP. Should I add all of them?